### PR TITLE
SDI-298 Add script to retrieve offender release dates

### PR DESCRIPTION
**Pull request by mikehalmamoj**
Captures the method used to retrieve the release dates of all offenders in the RP database.

If we have to do this on a regular basis then we should either create an endpoint to retrieve this information or build a UI.

### CODE REVIEW

**Review by hyderkhanmoj — APPROVED** [review]
LGTM

File: scripts/get-dates.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  if [[ -z "${RESULTS}" ]]; then
+    echo "Failed to receive a result for nomsNumber=${nomsNumber}, bookingId=${bookingId}"
+  fi
+
+  conditionalReleaseDate=$(echo $RESULTS | jq .conditionalReleaseDate)
+  licenceExpiryDate=$(echo $RESULTS | jq .licenceExpiryDate)
+  sentenceExpiryDate=$(echo $RESULTS | jq .sentenceExpiryDate)
+
+  echo "${nomsNumber},${bookingId},${conditionalReleaseDate},${licenceExpiryDate},${sentenceExpiryDate}" >> results.csv
+  sleep 1
```
Comment (andymarke) [inline scripts/get-dates.sh:25]: Can you do 0.5 ?

**Review by andymarke — APPROVED** [review]
Lgtm

File: scripts/get-dates.sh (hunk 1 above)
Comment (mikehalmamoj) [reply scripts/get-dates.sh:25]: I quite enjoyed the 15 minutes break tbh

**Pull request merged**